app/main.py

```
@@ -0,0 +1,555 @@
+            background_tasks.add_task(update_summary, chat_id, user_id, 40, 80, f"{user_id}-{chat_id}.medium_text_summary")
+            background_tasks.add_task(update_summary, chat_id, user_id, 80, 160, f"{user_id}-{chat_id}.long_text_summary")
+            background_tasks.add_task(redis.set, f"{user_id}-{chat_id}.is-active", "false")
+            return {"message": response_content}
+
+        background_tasks.add_task(redis.set, f"{user_id}-{chat_id}.is-active", "false")
+        return {"error": response.status_code}
+    except mariadb.Error as e:
+        background_tasks.add_task(redis.set, f"{user_id}-{chat_id}.chat_is-active", "false")
+        print(e)
+        return {"error": f"{e}"}
+    except Exception as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)